server.js

```diff
@@ -20,11 +13,38 @@ app.use(cors());
 app.use(express.json());
 
 // Start defining your routes here
-app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+app.get('/', (req, res) => {
+	res.send('Golden Globe Winners');
+});
+
+//Fetches data from the API//
+app.get('/goldenGlobes', (req, res) => {
+	res.status(200).json(goldenGlobesData);
+});
+
+//Films by year//
+app.get('/goldenGlobes/year_award/:year_award', (req, res) => {
+	const filmByYear = goldenGlobesData.find(
+		(goldenGlobes) => goldenGlobes.year_award === req.params.year_award
+	);
+	if (!goldenGlobesData) {
+		res.status(404).json('Not found');
+	} else {
+		res.status(200).json({ data: filmByYear, success: true });
+	}
```

Comment:
on the status 200 you have data and success, you could have that as well on the status 404 :
{data: "not found", 
success: false,
}